src/Magnum/Audio/Renderer.h

```diff
@@ -188,6 +240,21 @@ class Renderer {
 /** @debugoperatorclassenum{Magnum::Audio::Renderer,Magnum::Audio::Renderer::Error} */
 Debug MAGNUM_AUDIO_EXPORT operator<<(Debug debug, Renderer::Error value);
 
+/** @debugoperatorclassenum{Magnum::Audio::Renderer,Magnum::Audio::Renderer::DistanceModel} */
+Debug MAGNUM_AUDIO_EXPORT operator<<(Debug debug, Renderer::DistanceModel value);
+
+inline Vector3 Renderer::listenerPosition() {
```

Comment:
I think you can put these directly into the class definition, they aren't too long to be hidden below.